### PR TITLE
Migration to gatsby v5 fresh start

### PR DESCRIPTION
**Pull request by sharnik**
This is based off work started in https://github.com/brains-and-beards/open-brainsandbeards-com/pull/8. 

I've created a new Gatsby v5 TypeScript project and started porting components from Szymon's branch and fixing any issues that came up. If you want to track the progress of this process, you can do that in [a dedicated temporary repo](https://github.com/brains-and-beards/brains-and-beards-com-dupla). Currently I think it's ready to be merged as it's basically as functional as current version on brainsandbeards.com.

There are some more tweaks (and bugfixing) that we need to do, but those can be done separately. Full list is in our Notion, dropping a screenshot here for context:

CleanShot 2023-04-18 at 20 38 12@2x

It'd be great if you could test out the deployed PR preview whether there are any bugs that have not been found / tracked yet.

### CODE REVIEW

File: src/components/shared/layout/Navbar.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                <CloseSvg className="mobile-only" alt="close" />
+              </div>
+            </div>
+            <ul>
+              {[
+                { url: "/", label: "Home" },
+                { url: "/projects", label: "Projects" },
+                { url: "/about-us", label: "About us" },
+                { url: "/team", label: "Team" },
+                { url: "/jobs", label: "Careers" },
```
Comment (szymonkoper) [inline src/components/shared/layout/Navbar.tsx]: Something wrong with this redirect

> 404 Not found
> You just hit a route that doesn't exist... the sadness.

Comment (szymonkoper) [reply src/components/shared/layout/Navbar.tsx]: I went through all other links, including project case studies and they all look good apart from this "Careers" one. 👍

I have not checked blog posts yet

File: README.md [hunk 2]
```diff
@@ -1,14 +1,49 @@
-# Brains & Beards web page
```
Comment (szymonkoper) [inline README.md]: I think we should keep old `README.md` contents instead of initial one from new Gatsby project

File: package.json [hunk 3]
```diff
@@ -1,57 +1,52 @@
...
-    "format": "prettier --write 'src/**/*.js'"
+    "start": "gatsby develop",
+    "build": "gatsby build",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@mdx-js/react": "^2.3.0",
+    "gatsby": "^5.7.0",
```
Comment (szymonkoper) [inline package.json:22]: The newest is now 5.9.0 🌝

File: src/pages/team.tsx [hunk 4]
```diff
@@ -1,93 +1,97 @@
...
         <div className="content-page">
           <div className="big-part-yellow">
             <div className="content members-box">
-              {teamMembers.map((member) => (
+              {teamMembersWithImages.map((member) => (
                 <TeamMember {...member} key={member.name} />
               ))}
             </div>
             <div className="mobile-only team-swiper">
               <Swiper
```
Comment (szymonkoper) [inline src/pages/team.tsx]: On "Team" page on smartphone I see only Patryk.

In fact it shows a swiper, but not centered and without any arrows indicating it's possible to swipe it. 

Screenshot 2023-04-19 at 10 02 38

Comment (sharnik) [reply src/pages/team.tsx]: Fixed 👍

File: package.json (hunk 3 above)
Comment (sharnik) [reply package.json:22]: I'll take a break from updating Gatsby and leave it as-is :)

File: README.md (hunk 2 above)
Comment (sharnik) [reply README.md]: I've updated it 👍

File: src/components/shared/layout/Navbar.tsx (hunk 1 above)
Comment (sharnik) [reply src/components/shared/layout/Navbar.tsx]: I've extracted the navbar link rendering to a separate component. Now it works as expected, the code is a bit cleaner, and I learnt what's going on there. (Apparently Gatsby has special JS handling for Link elements and treats them as internal.)

**Review by szymonkoper — APPROVED** [review]

**Pull request merged**